server/konnectors/edf.coffee

```diff
@@ -1143,20 +1204,42 @@ displayData = (requiredFields, entries, data, next) ->
 
     next()
 
+fetchEdeliaData = (requiredFields, entries, data, next) ->
```

Comment:
Why don't you put these operations directly in the operation lit of the connector? Or in a new konnector?